release.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Borrowed from: https://github.com/informalsystems/tendermint-rs/blob/8ba855004b1f2d180e0c6e15f2f81436db3e9003/release.sh
+
+# release.sh will hopefully allow us to publish all of the necessary crates in
+# this repo in the right order. It is assumed that only one person will be
+# releasing all crates at the same time.
+#
+# It has a default set of crates it will publish, which can be overridden by
+# way of command line arguments:
+#
+#   # Release all packages, prompting for each package as to whether to publish
+#   ./release.sh
+#
+#   # Just release the proto and tendermint crates, but nothing else
+#   ./release.sh proto tendermint
+
+set -e
+
+# A space-separated list of all the crates we want to publish, in the order in
+# which they must be published. It's important to respect this order, since
+# each subsequent crate depends on one or more of the preceding ones.
+DEFAULT_CRATES="ibc"
```

Comment:
Not sure we need this whole script if only a single package is being published. One could easily, in the workflow file, simply run something like:

```bash
cargo publish -p ibc --token ${{secrets.CRATES_TOKEN}}
```

This script is necessary for tendermint-rs because of the crate interdependencies, and because after publishing a crate _and_ successfully confirming via the crates.io API that the new crate version has been published, subsequent crates that depend on it _still_ fail to be built sometimes for some reason (my guess is that they haven't managed to propagate through the CDN yet).

Reply:
I thought this was a good default, but maybe it's best to go for the one-liner indeed